feat(curriculum): add animations to block 3.2 of the English Curriculum (#54033)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-talk-about-hobbies-and-interests/657e76d866ebcec0e44d0748.md

title: Task 29
challengeType: 19
dashedName: task-29
---

<!-- (Audio) Jake: I haven't been on a bike since I was a kid. -->

# --description--

> In the Present Perfect Tense, to make a sentence negative, you add `not` after `have` or `has`. This way, you can talk about something that has not occurred lately. 

Reviewer: ```suggestion
In the Present Perfect Tense, to form a negative sentence, you insert `not` directly after `have` or `has`. This construction allows you to express actions or events that have not taken place recently.
```

> For example, if Jake wants to say it's been a long time since he was on a bike for the last time, he can say, `I have not been on a bike since I was a child.`.

Reviewer: ```suggestion
For example, if Jake wishes to express that a significant amount of time has passed since his last bicycle ride, he could say, `I have not been on a bike since I was a child.`
```

> In this sentence, you could also be abbreviated the verb `have`: `I haven't been on a bike since I was a child.`

Reviewer: ```suggestion
In this sentence, the verb `have` can also be abbreviated to `haven't`: `I haven't been on a bike since I was a child.`
```
